Exercise_sequential_fastqc/fastqc_arrayjob.slurm.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 #SBATCH --partition=reservation  # Use a regular partition such as "express" when not in training & remove --reservation 
-#SBATCH --reservation=fall_training_cpu_2022
+#SBATCH --reservation=bootcamp_cpu_2023
 #SBATCH --nodes 1
 #SBATCH --cpus-per-task=1
 #SBATCH -t 1:00:00
```

Comment:
It won't let me select the line since it wasn't edited it but I would remove your email address for the slurm emails.

Reply:
The job fails as well when you run it. Since fastqc version isn't specified, the latest is used and it is build with the OpenJDK instead of the oracle one.

So either
module load oracle_java/jdk1.8.0_181
module load fastqc/0.11.8

or

module load OpenJDK/19.0.1 
module load fastqc/0.11.9 or just leave it as module load fastqc

Reply:
@joseph-cooke, thanks for catching that! I've now added the version number and tested it out. It worked fine for me.